.nais/config-preprod-q2.json

```
@@ -0,0 +1,35 @@
+{
+	"appname": "soknadsarkiverer",
+	"profile": "dev",
+	"replicas-min": "2",
+	"replicas-max": "4",
+	"ingress": "https://soknadsarkiverer-gcp.intern.dev.nav.no",
+	"outboundRules": [
+		"soknadsfillager",
+		"innsending-api"
+	],
+	"outboundExternalHosts": [
+		"dokarkiv.dev-fss-pub.nais.io",
+		"soknadsfillager-gcp.intern.dev.nav.no",
+		"innsending-api-gcp.intern.dev.nav.no",
+		"saf.dev-fss-pub.nais.io"
```

Må ikke denne også oppdateres? 
```suggestion
		"saf-q2.dev-fss-pub.nais.io"
```